docs/content/examples/use-cases/external-relay-only-mailserver.md

```diff
@@ -0,0 +1,213 @@
+---
+title: 'Use Cases | Use a public server for relaying between a private DMS instance'
+hide:
+  - toc
+---
+
+## Introduction
+
+!!! info "Community contributed guide"
+
+    Adapted into a guide from [this discussion](https://github.com/orgs/docker-mailserver/discussions/3965).
+
+    **Requirements:**
+
+    - A _public server_ with a static IP, like many VPS providers offer. It will only relay mail to DMS, no mail is stored on this system.
+    - A _private server_ (eg: a local system at home) that will run DMS.
+    - Both servers are connected to the same network via a VPN (_optional convenience for trust via the `mynetworks` setting_). We will assume below that the VPN is setup on `192.168.2.0/24`, with the _public server_ using `192.168.2.2` and the _private server_ using `192.168.2.3`.
+
+The goal of this guide is to configure a _public server_ that can receive inbound mail and relay that over to DMS on a _private server_, which can likewise submit mail outbound through a _public server_ or service. The primary motivation is keep your mail storage private, instead of storing unencrypted on a VPS host disk.
```

Comment:
```suggestion
The goal of this guide is to configure a _public server_ that can receive inbound mail and relay that over to DMS on a _private server_, which can likewise submit mail outbound through a _public server_ or service.

The primary motivation is to keep your mail storage private, instead of storing unencrypted on a VPS host disk.
```